pom.xml

```diff
@@ -10,12 +10,13 @@
 		<name>Pivotal Software, Inc.</name>
 		<url>http://www.spring.io</url>
 	</organization>
-    <properties>
-      <java.version>1.7</java.version>
-      <spring-boot.version>1.3.0.BUILD-SNAPSHOT</spring-boot.version>
-      <spring-cloud.version>Brixton.BUILD-SNAPSHOT</spring-cloud.version>
-      <spring-xd.version>2.0.0.BUILD-SNAPSHOT</spring-xd.version>
-    </properties>
+	<properties>
+		<java.version>1.7</java.version>
+		<spring-boot.version>1.3.0.BUILD-SNAPSHOT</spring-boot.version>
+		<spring-cloud.version>Brixton.BUILD-SNAPSHOT</spring-cloud.version>
+		<spring-xd.version>2.0.0.BUILD-SNAPSHOT</spring-xd.version>
+		<spring.framework.version>4.2.0.RC2</spring.framework.version>
```

Comment:
why not use spring-framework (instead of the dot delimiter) for consistency with the others?